srtcore/stats.h

```diff
@@ -167,6 +170,18 @@ struct Sender
         recvdNak.resetTrace();
         sentFilterExtra.resetTrace();
     }
+
+    void fulfillMeasuresTable(int pkts, double bytes) {
+        mobileRateEstimator.addSample(pkts, bytes);
+    }
+
+    void resetMeasuresTable() {
+        mobileRateEstimator.resetMeasuresTable();
+    }
+
+    int getAverageValueFromTable(){
```

Comment:
Maybe rename to `getAverateRateMbps`? The "from table" part is an implementation specifics a user of stats does not need to be aware about.
The function can be marked with `const`.

Reply:
Done.